src/handlers/RF24StreamLogHandler.cpp

```diff
@@ -149,45 +153,45 @@ void RF24StreamLogHandler::appendLogLevel(uint8_t logLevel)
 
     switch (logMainLevel)
     {
-    case RF24LogLevel::ERROR:
-    {
-        stream->print(rf24LogLevels[0]);
-        break;
-    }
-    case RF24LogLevel::WARN:
-    {
-        stream->print(rf24LogLevels[1]);
-        break;
-    }
-    case RF24LogLevel::INFO:
-    {
-        stream->print(rf24LogLevels[2]);
-        break;
-    }
-    case RF24LogLevel::DEBUG:
-    {
-        stream->print(rf24LogLevels[3]);
-        break;
-    }
-    default:
-    {
-        // unknown
-        if (logLevel < 10)
+        case RF24LogLevel::ERROR:
         {
-            stream->print("Lvl   ");
+            stream->print(rf24LogLevels[0]);
+            break;
         }
-        else if (logLevel < 100)
+        case RF24LogLevel::WARN:

```

Comment:
I didn't consciously change the `if ... else if ... else` to `switch ... case`. No idea what happened here. @wmarkow I've never seen curly brackets used in `case \<val>:`  code blocks, but I think its from whatever changed the `if ... else if ... else` to `switch ... case`. I vaguely remember that your testing showed no advantage to using `if` statements vs a `switch` block.

Reply:
we could use `stream->print(rf24LogLevels[(logLevel >> 3) - 1]);` instead of a switch block.

Reply:
Moving this discussion to #8